src/test/unit_tests/test_nb_native_fs.js

```diff
@@ -12,9 +13,9 @@ const { get_process_fs_context } = require('../../util/native_fs_utils');
 
 const DEFAULT_FS_CONFIG = get_process_fs_context();
 
-mocha.describe('nb_native fs', function() {
+mocha.describe('nb_native fs', async function() {
```

Comment:
General question - why do you add the `async` in the describe part? did you see an issue with this?
From what I saw in the docs the function inside `it` is async, but the `describe` doesn't contain it (see [USING ASYNC / AWAIT](https://mochajs.org/#using-async-await))